examples/custom-navigation/call-activity-with-modal-on-mouse-over/index.js

```diff
@@ -14,10 +14,6 @@ const modalElt = document.getElementById('modal');
 let secondaryBpmnDiagramIsAlreadyLoad = false;
 const callActivityElt = mainBpmnVisualization.bpmnElementsRegistry.getElementsByIds(['call_activity'])[0].htmlElement;
 
-// WARNING: The class adding doesn't work with the zoom, the panning and the fit
-// TODO Need to replace by the dedicated API when it is implemented
-callActivityElt.classList.add('c-hand');
```

Comment:
Why do you completly remove this css class ?

If there is a problem with the modal, the user doesn't know where is the location where it should have a behaviour.

Reply:
ah, it was unclear to me why we needed this 'hand' cursor as it never displays when everything is working.
I am going to restore it with a comment explaining it.